lib/helpers/property-exists.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function checkNestedProperty (obj /*, ...Properties*/) {
+  var args = Array.prototype.slice.call(arguments, 1);
+
+  for (var i = 0; i < args.length; i++) {
+    if (!obj || !obj.hasOwnProperty(args[i])) {
+      return false;
+    }
+    obj = obj[args[i]];
+  }
+  return true;
+};
```

Comment:
Minor, but I don't think this needs to go in a helpers folder - it's essentially what the `lib` folder is!